docs/08-add-component.md

```diff
@@ -0,0 +1,67 @@
+# Adding a component to the stack.
+
+## Vanilla
+
+---
+
+The vanilla version of the component, so it's a node package made of a style file and a javascript when needed, which exports a mixin of the component styles and the related javascript as a module.
+The name of the component is prefixed by _{system}-component_
+
+### Utilities packages
+
+There is a set of packages in the list of the vanilla ones which are providing utilities classes to be applied to the components when needed. Their names are prefixed by _{system}-utility_
```

Comment:
We have to make it clear in the developer doc that utilities are to be used when building layout, but the components themselves should not use utilities

Reply:
ok, thanks for the clarification, it is also good for me to know that, now :)